scripts/automatic_xml_preprocess.sh

```diff
@@ -1,13 +1,16 @@
 #!/bin/bash
 
-# Parse and modify input arguments
+
+# Parse input arguments
 INPUT_ARGS=""
 INPUT_COUNTER=0
 COMPILED_XML_NAME=""
 COMPILED_XML_NAME_OVERRIDE=""
 PARAMETER_ARGS=""
 NEW_ARGS=""
+USE_PYGEOSX=1
```

Comment:
If we would like to require pygeosx to be installed, then this script could be greatly simplified

Reply:
I haven't been unable to build pygeosx so far, because it hard codes python version 3.8 and our cluster only has 3.6

https://github.com/GEOSX/GEOSX/blob/1414a7dc5d572b5852bcd5654f3c5e3954ffeec3/src/pygeosx/CMakeLists.txt#L25

Reply:
For pygeosx to work properly, you need to build a version of python (and certain dependencies) with exact compiler / mpi matches.  I used spack to do so for the Lassen build